doc/faq.md

```diff
@@ -176,6 +176,29 @@ re-install Convos:
 
 Please [contact us](/doc/#get-in-touch) if the above instructions do not work.
 
+## Why convos does not restart after an upgrade of convos code?
+
+Some changes in the convos code do not require any special operation (for instance if you modify only assets and/or themes). 
```

Comment:
Could this be more specific? "Like changes in the `public/` directory does not require you to restart Convos"?

Reply:
I tried to be more specific without being exhaustive (because I don't know what is the exact list of things that we can change without having to restart convos services). Is it better for you?

Reply:
Oh! I was wrong: The compiled assets (css+js) will be in public/asset/, and requires you to restart Convos. Maybe it would be better to just say that after you update (git pull, or whatever), you need to restart Convos. If you want to modify themes while Convos is running, then modify them in `$CONVOS_HOME/themes`.

Reply:
Again, this is why I think it might be better with an upgrade guide.

Reply:
Ok, what if we just close this PR ? 

There are some problems that are not easy to fix without a major rework.

It could just serve as an inspiration guide when someone will write this updgrade guide :smile: